core/src/apps/ethereum/layout.py

```diff
@@ -109,16 +110,26 @@ def require_confirm_data(ctx: Context, data: bytes, data_total: int) -> Awaitabl
     )
 
 
-async def confirm_hash(ctx: Context, message_hash: bytes) -> None:
-    await confirm_blob(
+async def confirm_typed_data_final(ctx: Context) -> None:
+    await confirm_action(
         ctx,
-        "confirm_hash",
-        title="Confirm hash",
-        data="0x" + hexlify(message_hash).decode(),
+        "confirm_typed_data_final",
+        title="Confirm typed data",
+        action="Confirm signing\nEIP-712 typed data?",
         hold=True,
     )
 
 
+def require_confirm_empty_typed_message(ctx: Context) -> Awaitable[None]:
```

Comment:
please rename to just `confirm_...` (the `require_` is an older naming convention that is not used anymore, everything is "require" nowadays)

Reply:
Fixed in https://github.com/trezor/trezor-firmware/pull/2036/commits/ec280a2cab209f3c6700541b3daf0efaafdd7b42

For some reason, the fixtures seemed to have changed too, but maybe I misrecorded them in the earlier commit (e.g. by running the emulator with animations on)
![image](https://user-images.githubusercontent.com/19716675/149361065-e1f8396a-6cc1-4932-957f-e51671c8f789.png)